ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -197,21 +198,19 @@ export default class ConfirmPageContainerContent extends Component {
           )}
         {showInsuffienctFundsError && (
           <div className="confirm-page-container-content__error-container">
-            {currentTransaction.chainId === MAINNET_CHAIN_ID ? (
+            {isBuyableTransakChain ? (
               <ActionableMessage
                 className="actionable-message--warning"
                 message={
                   <Typography variant={TYPOGRAPHY.H7} align="left">
                     {t('insufficientCurrency', [nativeCurrency, networkName])}
                     <Button
-                      key="link"
-                      type="secondary"
+                      type="link"
```

Comment:
Type inline might be of more use here?
```suggestion
                      type="inline"
```